packages/cms/pages/assests/asset-manager/asset-manager.tsx

```diff
@@ -239,28 +264,30 @@ export const AssetManager: FunctionComponent = () => {
             <EuiTitle size="xs">
               <h3>Created by</h3>
             </EuiTitle>
-            <EuiText>John Doe</EuiText>
+            <EuiText>{active?.name}</EuiText>
           </FlyoutBodyContent>
 
           <FlyoutBodyContent>
             <EuiTitle size="xs">
               <h3>Size</h3>
             </EuiTitle>
-            <EuiText>2.3MB</EuiText>
+            <EuiText>{active && (active.size / 1000).toFixed(1)}MB</EuiText>
```

Comment:
Let's create a function that converts to relevant size formats. I think the old codebase has this